src/polyfem/state/StateInit.cpp

```diff
@@ -296,7 +296,8 @@ namespace polyfem
 		}
 
 		// Save output directory and resolve output paths dynamically
-		const std::string output_dir = this->args["output"]["directory"];
+		const std::string output_dir = resolve_input_path(this->args["output"]["directory"]);
```

Comment:
Does this mean even the command line outout directory option has to be relative to the input JSON?

Reply:
yes, is this a bad thing?

Reply:
I like to specify -o outputdir/blah knowing the output will be placed in the current working directory. An easy fix is to resolve the relative path in the main so it becomes an absolute path.

Reply:
this will have a different effect if you use `-o` vs the stuff in the json, I like it

Reply:
fixed in 27e7a6dd5df537c3f54cc3bc20db9bbaad72f815

Reply:
Awesome, ship it!